test/integration/ProjectBrowser.kt

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
Rename all integration test files to `*Test.kt`, all classes to `*Test`.

Reply:
This has been done.